src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -56,6 +57,10 @@ const defaultValues: FormValues = {
   [ListingField.SLEEPING_ARRANGEMENT]: '',
   [ListingField.STATE]: '',
   [ListingField.TITLE]: '',
+  [ListingField.WIFI]: {
+    photo: '',
+    speed: 0,
```

Comment:
we should have the measurement unit as part of the name

Reply:
changed `speed` to `mbps`